ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -215,6 +215,9 @@ objects:
         chmod +x "${HOME}"/run-tests.sh
         mock-nss.sh
         mkdir "${HOME}"/.ssh
+        # gcloud compute will use this key rather than create a new one
+        cp -a /tmp/cluster/ssh-privatekey "${HOME}"/.ssh/google_compute_engine
+        cp -a /tmp/cluster/ssh-publickey "${HOME}"/.ssh/google_compute_engine.pub
```

Comment:
nit: `"${HOME}"` -> `~`?  Although I guess there's precedent above for the chatty version...

Reply:
updated